<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow3</class>
 <widget class="QMainWindow" name="MainWindow3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>742</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1204</width>
    <height>742</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1204</width>
    <height>742</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;
background-image: url(:/new/prefix1/resourse/startpage/1.jpg);
background-color: rgb(255, 161, 161);

</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>640</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;
background-color: rgb(255, 161, 161);
background-image: url(:/new/prefix1/resourse/startpage/1.jpg);</string>
    </property>
    <property name="text">
     <string>全球动态</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>640</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;
background-color: rgb(255, 161, 161);
background-image: url(:/new/prefix1/resourse/startpage/1.jpg);</string>
    </property>
    <property name="text">
     <string>个人动态</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>640</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;
background-color: rgb(255, 161, 161);
background-image: url(:/new/prefix1/resourse/startpage/1.jpg);</string>
    </property>
    <property name="text">
     <string>关于我</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_5">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>640</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;
background-color: rgb(255, 161, 161);
background-image: url(:/new/prefix1/resourse/startpage/1.jpg);</string>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/resourse/startpage/1.jpg);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>741</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 75 24pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(:/new/prefix1/resourse/startpage/1.jpg);
color: rgb(0, 85, 127);
font: 75 24pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>探寻！</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="b1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="b2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>1211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="b3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>1211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="b4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>1211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_6">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>50</y>
      <width>187</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>发布动态</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
